e2e-polybft/txpool_test.go

```diff
@@ -159,6 +161,36 @@ func TestE2E_TxPool_Transfer_Linear(t *testing.T) {
+
+	customArtifact, err := artifact.DecodeArtifact([]byte(smJSON))
+	require.NoError(t, err)
+
+	sidechainKey, err := wallet.GenerateKey()
+	require.NoError(t, err)
+
+	cluster := framework.NewTestCluster(t, 4,
+		framework.WithPremine(types.Address(sidechainKey.Address())),
+	)
+	defer cluster.Stop()
+
+	require.NoError(t, cluster.WaitForBlock(5, 30*time.Second))
+
+	relayer, err := txrelayer.NewTxRelayer(txrelayer.WithIPAddress(cluster.Servers[0].JSONRPCAddr()))
+	require.NoError(t, err)
+
+	receipt, err := relayer.SendTransaction(&ethgo.Transaction{Input: customArtifact.Bytecode}, sidechainKey)
+	require.NoError(t, err)
+	require.Equal(t, uint64(types.ReceiptSuccess), receipt.Status)
+
+	receipt, err = ABITransaction(relayer, sidechainKey, customArtifact, receipt.ContractAddress, "init", []interface{}{})
+	require.NoError(t, err)
+	require.Equal(t, uint64(types.ReceiptSuccess), receipt.Status)
+
+	require.NoError(t, cluster.WaitForBlock(25, 2*time.Minute))
```

Comment:
Maybe you don't need to wait so long here? 

I'd say it would be enough to:
1. shorten epoch size to let's say 5 blocks
2. wait for block 10

Reply:
size of epoch really does not matters for a test, but changed to wait for 10  https://github.com/0xPolygon/polygon-edge/pull/1140/commits/cb1fa0a0415784fa391b7dd3961651021f9d934a